src/main/java/com/shift4/enums/FlowNextAction.java

```diff
@@ -0,0 +1,38 @@
+package com.shift4.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum FlowNextAction {
```

Comment:
To be consistent with current naming: ChargeFlowActionType -> PaymentMethodFlowActionType